### PR TITLE
added card component

### PR DESCRIPTION
**Pull request by irajbhinde**
Hey! Please review my code & let me know if any changes could be made.

Added Components -

**Cards**

1. Cards with badges
2. Cards with dismiss icon
3. Cards with Text Overlay
4. Text only card
5. Horizontal Cards
6. Vertical Cards With Shadow

**GIFY** -

cards

### CODE REVIEW

File: Components/Cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  width: min-content;
+  height: auto;
+  box-shadow: rgb(50 50 93 / 25%) 0px 2px 5px -1px,
+    rgb(0 0 0 / 30%) 0px 1px 3px -1px;
+  padding: 1rem;
+  flex-wrap: wrap ;
+}
+
+.cards.textonly, .cards.horizontal{
+  width: 350px;
```
Comment (akash-1318) [inline Components/Cards/cards.css]: use rem instead of pixel here.

File: Components/Cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+.cards.textonly {
+  box-shadow: rgb(50 50 93 / 25%) 0px 2px 5px -1px,
+    rgb(0 0 0 / 30%) 0px 1px 3px -1px;
+  background-color: white;
+}
+
+.text-only-title {
+  padding: 0.5rem 1rem 0.5rem;
+  font-size: 25px;
```
Comment (akash-1318) [inline Components/Cards/cards.css]: Here also you can use rem

**Review by akash-1318 — COMMENTED** [review]
Overall code looks really clean just avoid using pixel unit if it's more then 10px.

File: Components/Cards/cards.css [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  padding: 1rem;
+  flex-wrap: wrap ;
+}
+
+.cards.textonly, .cards.horizontal{
+  width: 350px;
+}
+
+.cards.textonly {
+  box-shadow: rgb(50 50 93 / 25%) 0px 2px 5px -1px,
```
Comment (rahul31agrawal) [inline Components/Cards/cards.css:17]: You can use Hex codes here

File: Components/Cards/cards.css [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  font-size: 12px;
+  position: absolute;
+  padding: 0.3rem;
+  width: max-content;
+}
+
+.top-right-badge {
+  right: 1px;
+  top: 20px;
+  color: white;
```
Comment (rahul31agrawal) [inline Components/Cards/cards.css:67]: you are using variables for colors so to maintain consistency you can use "var color" here also.

**Review by rahul31agrawal — APPROVED** [review]

**Comment by rahul31agrawal** [comment]
Nice work Raj, looks good to merge.
thanks

**Pull request merged**